### ISSUE TITLE
Add shortcuts to put tiles in the 1st layer or 2nd or 3rd layer.

### ISSUE THREAD

**Issue by maxamine** [christopho/solarus-quest-editor#93]
in addition to the (1, 2, 3) shortcuts to display layers, it would be useful to have shortcuts (Ctrl +1, Ctrl + 2, CTRL + 3) respectively to put the tiles in the first layer or second or the third layer.

**Comment by christopho** [comment]
In the editor < 1.4, there were actually shortcuts + and - to move entities one layer up or one layer down.
Ctrl-1/2/3 could be nice too.

**Comment by Renkineko** [comment]
The Ctrl+1/2/3 could be useless soon if you stop the 3 layers limitation. On a map with more than 10 layers, what Ctrl+number use ? IMHO, only the +/- can be useful : on a 3 layers limited map, it's the same delay to push twice + than do ctrl + 3 ; on an unlimited layer map, you get more control by using only +/-.

**Comment by maxamine** [comment]
i agree with you Renkineko but when you work on a dungeon for exemple and you have a map which contains more than 1000 tiles, +/- allows you to move entities one layer up or down, but you can't put directly some tiles in the 2nd layer, as you say if there were 10 layers in the map CTRL+2 for exemple allows to put them directly in the desired layer.

**Comment by christopho** [comment]
We will put both shortcuts.

**Comment by maxamine** [comment]
ok christopho good idea

**Comment by christopho** [comment]
+/- are now done in https://github.com/christopho/solarus-quest-editor/commit/ed6fd01e315e31f5e312820e3a647cfe8de84d76
